### ISSUE TITLE
Brew autocomplete doesn't show up

### ISSUE THREAD

**Issue by peterjbachman** [withfig/fig#796]
### Description:
> Please include a detailed description of the issue (and an image or screen recording, if applicable)

The autocomplete popup does not show up for brew commands when using Fig, but other autocomplete functions appear to work. I've tried restarting and also reinstalling Fig. I installed Fig through Homebrew if that's important.

https://user-images.githubusercontent.com/46987067/150054136-b31e0ddc-8275-4a80-8826-09c1ce3b5e29.mov

Let me know if there's anything else you need from me!

### Details:
|macOS|Fig|Shell|
|-|-|-|
|12.1 21C52|1.0.54|/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Fig Version: Version 1.0.54 (B359) [U.S.] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /usr/local/bin/fig
Accessibility: true
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path: 
iTerm Integration: installed!
Hyper Integration: application is not present.
VSCode Integration: application is not present.
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installed via Brew: true
Installation Script: true
PseudoTerminal Path: /usr/local/opt/grep/libexec/gnubin:/Users/peter/.emacs.d/bin:/Users/peter/.cargo/bin:/Users/peter/.local/bin:/Users/peter/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/usr/local/MacGPG2/bin:/opt/X11/bin:/Users/peter/.fig/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: zsh (16997) - /dev/ttys004
Current working directory: /Users/peter
Current window identifier: 3933/% (com.googlecode.iterm2)
Path: /usr/local/opt/grep/libexec/gnubin:/Users/peter/.emacs.d/bin:/Users/peter/.cargo/bin:/Users/peter/.local/bin:/Users/peter/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/usr/local/MacGPG2/bin:/opt/X11/bin:/Users/peter/.fig/bin
Fig environment variables:
  - TERM_SESSION_ID=w1t0p0:4F3C95AA-B14B-42AA-AF25-8A8ABFE8B663
  - FIG_INTEGRATION_VERSION=5
  - FIG_TERM=1
  - FIG_TERM_VERSION=3
  - FIG_ENV_VAR=1
  - FIG_CHECKED_PROMPTS=1
MacOS Version: 12.1 21C52
Hardware:
  - Model Name: MacBook Pro
  - Model Identifier: MacBookPro16,2
  - Chip: 
  - Cores: 4
  - Memory: 16 GB</pre>
</p>
</details>

**Comment by mschrage** [comment]
First off @peterjbachman, thanks for adding this screen recoding! It helps diagnose what is going on so much quicker.

Can you run `which brew`? My guess is that you have some alias/function defined that Fig isn't able to interpret.

PS. love the terminal theme + your desktop background. So ~aesthetic~

**Comment by peterjbachman** [comment]
Happy to help the process go smoother! Also thanks for noticing my setup!

So I blanked checking my aliases configuration, and when I commented out the one I had for `brew` it cleared up immediately.

Here's what I had for the output `which brew` if you're curious:

`brew: aliased to env PATH="${PATH//$(pyenv root)\/shims:/}" brew`

Dunno why I had that in the first place but it has now been removed. 🤷‍♂️
